packages/utils/gnodao/deploy.ts

```diff
@@ -74,35 +71,33 @@ const generateDAORealmSource = (networkId: string, conf: GnoDAOConfig) => {
       },
     }
   
-    messageHandlersFactories := []dao_interfaces.MessageHandlerFactory{
-      func(core dao_interfaces.IDAOCore) dao_interfaces.MessageHandler {
-        return groups.NewAddMemberHandler()
-      },
-      func(core dao_interfaces.IDAOCore) dao_interfaces.MessageHandler {
-        return groups.NewDeleteMemberHandler()
-      },
-      func(core dao_interfaces.IDAOCore) dao_interfaces.MessageHandler {
-        // TODO: add a router to support multiple proposal modules
-        propMod := core.ProposalModules()[0]
-        return proposal_single.NewUpdateSettingsHandler(propMod.Module.(*proposal_single.DAOProposalSingle))
-      },
-      func(core dao_interfaces.IDAOCore) dao_interfaces.MessageHandler {
-        return modboards.NewCreateBoardHandler()
-      },
-      func(core dao_interfaces.IDAOCore) dao_interfaces.MessageHandler {
-        return modboards.NewDeletePostHandler()
-      },
-    }
-  
-    daoCore = dao_core.NewDAOCore(votingModuleFactory, proposalModulesFactories, messageHandlersFactories)
+	messageHandlersFactories := []dao_interfaces.MessageHandlerFactory{
+		func(core dao_interfaces.IDAOCore) dao_interfaces.MessageHandler {
+			return group.UpdateMembersHandler()
+		},
+		func(core dao_interfaces.IDAOCore) dao_interfaces.MessageHandler {
+			// TODO: add a router to support multiple proposal modules
+			propMod := core.ProposalModules()[0]
+			return proposal_single.NewUpdateSettingsHandler(propMod.Module.(*proposal_single.DAOProposalSingle))
+		},
+		func(core dao_interfaces.IDAOCore) dao_interfaces.MessageHandler {
+			return tori.NewMintToriHandler()
+		},
+		func(core dao_interfaces.IDAOCore) dao_interfaces.MessageHandler {
+			return tori.NewBurnToriHandler()
+		},
```

Comment:
normal daos don't need to (and can't) control the tori coin, it's only in the demo dao